### PR TITLE
Fix for several fatal errors

### PR DESCRIPTION
**Pull request by annando**
Some stuff to fix several fatal errors.

### CODE REVIEW

File: src/Content/OEmbed.php [hunk 1]
```diff
@@ -86,8 +86,7 @@ public static function fetchURL($embedurl, $no_rich_type = false)
 				$redirects = 0;
 				$html_text = Network::fetchUrl($embedurl, false, $redirects, 15, 'text/*');
 				if ($html_text) {
-					$dom = new DOMDocument();
-					$dom->loadHTML($html_text);
+					$dom = @DOMDocument::loadHTML($html_text);
```
Comment (MrPetovan) [inline src/Content/OEmbed.php]: This isn't a static method, what was the fatal error you fixed with this?

**Comment by MrPetovan** [comment]
Related to #6386

File: src/Content/OEmbed.php (hunk 1 above)
Comment (annando) [reply src/Content/OEmbed.php]: I got massive warnings because of malformatted HTML - so I reverted the code here.

Comment (annando) [reply src/Content/OEmbed.php]: No. This had been my first move.

Comment (MrPetovan) [reply src/Content/OEmbed.php]: Drat.

**Pull request merged**